tools/github-bot/src/features/upToDate/index.ts

```diff
@@ -64,6 +64,7 @@ export function upToDate(app: Probot) {
     }
   );
 
+  /** @ts-expect-error it seems TypeScript have issues inferring the types here... */
```

Comment:
@valpinkman i have no clue about this part.

Reply:
Yup forget about it it's not important for this part of the code